.github/workflows/build-and-test-distributions.yml

```diff
@@ -0,0 +1,152 @@
+name: Build and test ofrak packages
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+    inputs:
+
+permissions:
+  contents: read
+
+defaults:
+  run:
+    shell: bash -euxo pipefail {0}
+
+jobs:
+  build-ofrak-type:
+    name: Build ofrak-type sdist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/build-sdist
+        with:
+          package-name: ofrak-type
+          source-dir: ofrak_type
+          python-version: '3.9'
+
+  build-ofrak-io:
+    name: Build ofrak-io sdist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/build-sdist
+        with:
+          package-name: ofrak-io
+          source-dir: ofrak_io
+          python-version: '3.9'
+
+  build-ofrak-patch-maker:
+    name: Build ofrak-patch-maker sdist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/build-sdist
+        with:
+          package-name: ofrak-patch-maker
+          source-dir: ofrak_patch_maker
+          python-version: '3.9'
+
+  build-pytest-ofrak:
+    name: Build pytest-ofrak sdist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/build-sdist
+        with:
+          package-name: pytest-ofrak
+          source-dir: pytest_ofrak
+          lfs: true
+          python-version: '3.9'
+
+  build-ofrak-angr:
+    name: Build ofrak-angr sdist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/build-sdist
+        with:
+          package-name: ofrak-angr
+          source-dir: disassemblers/ofrak_angr
+          python-version: '3.9'
+
+  build-ofrak-capstone:
+    name: Build ofrak-capstone sdist
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/build-sdist
+        with:
+          package-name: ofrak-capstone
+          source-dir: disassemblers/ofrak_capstone
+          python-version: '3.9'
```

Comment:
I think I commented this elsewhere, but should the Python version just be a default value instead of being repeated over and over?